packages/appeals-service-api/src/lib/notify.js

```diff
@@ -116,6 +116,33 @@ async function sendEmail(appeal) {
   }
 }
 
+async function sendAppealSubmissionConfirmationEmailToAppellant(appeal) {
+  if (!isValidAppealForSubmissionReceivedNotificationEmail(appeal)) {
```

Comment:
should this be

`isValidAppealForSubmissionConfirmationEmailToAppellantEmail`

or have I misunderstood ? (genuinely unsure)

Reply:
To be honest, I just resued the same method to check the state of the appeal before sending as in existing `sendAppealSubmissionReceivedNotificationEmailToLpa`. Agree its a poorly named method and maybe the check is a little blunt, but just blindly followed the exising pattern if Im honest.